pallets/connectors/src/message.rs

```diff
@@ -48,19 +49,14 @@ where
 	Transfer {
 		pool_id: PoolId,
 		tranche_id: TrancheId,
-		domain: Domain,
-		destination: Address,
+		address: Address,
 		amount: Balance,
+		domain: Domain,
```

Comment:
This is a minor detail, but semantically I don't think it makes sense that `domain` is last. This should at least be before `address`, wdyt?

I would possibly also prefix these to make it clearer, so
```rust
Transfer {
		pool_id: PoolId,
		tranche_id: TrancheId,
		destination_domain: Domain,
		destination_address: Address,
		amount: Balance,
	}
```

Reply:
I think that makes sense, I thought about this only after we set this order in Solidity 🙈  Let's get this right now 👍

Reply:
Done 👍